lego/apps/users/models.py

```diff
@@ -487,6 +487,30 @@ def unanswered_surveys(self) -> list:
         )
         return list(unanswered_surveys.values_list("id", flat=True))
 
+    def check_for_deletable_penalty(self):
```

Comment:
Could you call both this and delete_and_adjust_future_activation_times() on the penalty object for check_for_expired_penalty() and expire_and_adjust_future_activation_times()

Reply:
You could maybe also add a comment explaing that this is just expiration because of 6 events passing, not because the penalty expiration_date has passed?